### ISSUE TITLE
Cookies

### ISSUE THREAD

**Issue by AlfaJackal** [Hari-Nagarajan/fairgame#726]
**Describe the bug**
Cookie layer shows up after several hours of using fairgame.

**Expected behavior**
"Accept Cookies" should be clicked automatically

**Screenshots**
![image](https://user-images.githubusercontent.com/17096950/115963855-9bb89080-a521-11eb-827b-a66df6725424.jpeg)

**Desktop:**
 - OS: Windows 10
 - Chrome
 - Version 90.0.4430.85

**Comment by virus0190** [comment]
Yes, same to me. After several hours the cookie question will appear again. It can't click anymore.
Can you fix this please?

**Comment by r1di** [comment]
Indirect solution is to install the browser plugin "**[i dont care about cookies](https://chrome.google.com/webstore/detail/i-dont-care-about-cookies/fihnjjcciajhdojfnbdddfaoknhalnja)**", this will fix it.
